redfish-core/lib/power_subsystem.hpp

```diff
@@ -25,6 +25,26 @@ namespace redfish
 inline void getPowerSubsystemAllocation(
     const std::shared_ptr<bmcweb::AsyncResp>& asyncResp)
 {
+    // Get max from CapLimit
+    sdbusplus::asio::getProperty<uint32_t>(
+        *crow::connections::systemBus, "org.open_power.OCC.Control",
+        "/xyz/openbmc_project/control/host0/power_cap_limits",
+        "xyz.openbmc_project.Control.Power.CapLimits", "MaxPowerCapValue",
+        [asyncResp](const boost::system::error_code& ec, uint32_t maxCap) {
+            if (ec)
+            {
+                if (ec.value() != EBADR)
+                {
+                    BMCWEB_LOG_ERROR("DBUS response error: {}", ec.value());
+                    messages::internalError(asyncResp->res);
+                }
+                return;
+            }
+            asyncResp->res.jsonValue["Allocation"]["AllocatedWatts"] = maxCap;
```

Comment:
On further reflection I realize this won't be returning a consistent value for "AllocatedWatts". Both of the places which are filling in the value are asynchronous handlers. So there is no guarantee on the order that the value will be filled in.

Before I was thinking this was just an issue for the error path. But I actually believe it is an issue for the normal path as well. I do think these calls are going to have to be coordinated to assure a consistent value is returned.